docs/frequently-asked-questions.rst

```diff
@@ -182,99 +29,16 @@ arguments for you.
-============================================================================================================
-
-This is not possible, because all types use up the full value range.
-
-You have the option to ``throw`` on error, which will also revert the whole
-transaction, which might be a good idea if you ran into an unexpected
-situation.
-
-If you do not want to throw, you can return a pair::
-
-    pragma solidity >0.4.23 <0.5.0;
-
-    contract C {
-        uint[] counters;
-
-        function getCounter(uint index)
-            public
-            view
-            returns (uint counter, bool error) {
-                if (index >= counters.length)
-                    return (0, true);
-                else
-                    return (counters[index], false);
-        }
-
-        function checkCounter(uint index) public view {
-            (uint counter, bool error) = getCounter(index);
-            if (error) {
-                // ...
-            } else {
-                // ...
-            }
-        }
-    }
-
-
-Are comments included with deployed contracts and do they increase deployment gas?
-==================================================================================
-
-No, everything that is not needed for execution is removed during compilation.
-This includes, among others, comments, variable names and type names.
-
-What happens if you send ether along with a function call to a contract?
-========================================================================
-
-It gets added to the total balance of the contract, just like when you send ether when creating a contract.
-You can only send ether along to a function that has the ``payable`` modifier,
-otherwise an exception is thrown.
-
-Is it possible to get a tx receipt for a transaction executed contract-to-contract?
-===================================================================================
-
-No, a function call from one contract to another does not create its own transaction,
-you have to look in the overall transaction. This is also the reason why several
-block explorer do not show Ether sent between contracts correctly.
 
 What is the ``memory`` keyword? What does it do?
 ================================================
 
-The Ethereum Virtual Machine has three areas where it can store items.
-
-The first is "storage", where all the contract state variables reside.
-Every contract has its own storage and it is persistent between function calls
-and quite expensive to use.
-
-The second is "memory", this is used to hold temporary values. It
-is erased between (external) function calls and is cheaper to use.
-
-The third one is the stack, which is used to hold small local variables.
-It is almost free to use, but can only hold a limited amount of values.
-
-For almost all types, you cannot specify where they should be stored, because
-they are copied everytime they are used.
-
 The types where the so-called storage location is important are structs
```

Comment:
I moved some content here, but this starts to look like quite a large topic, so I wonder how much is relevant?